digital-workplace-webapps/src/main/webapp/vue-app/topBar/notification/components/NotificationApp.vue

```diff
@@ -24,102 +24,128 @@
+          <v-row class="mx-0 px-3">
+            <v-list-item class="notifDrawerHeader">
+              <v-list-item-content>
+                <span class="notifDrawerTitle">{{ this.$t('homepage.topBar.notification') }}</span>
+              </v-list-item-content>
+              <v-list-item-action class="notifDrawerIcons">
+                <i class="uiSettingsIcon notifDrawerSettings mr-3"></i>
+                <i class="uiCloseIcon notifDrawerClose" @click="closeDrawer()"></i>
+              </v-list-item-action>
+            </v-list-item>
+          </v-row>
 
-                    <v-list-item
-                      v-for="(item, i) in items"
-                      :key="i">
-                      <v-layout :style="{background:item.background}">
-                        <v-flex xs2>
-                          <v-card style="display: contents">
-                            <v-list-item-avatar class="ml-4">
-                              <v-img :src="item.avatar"/>
-                            </v-list-item-avatar>
-                          </v-card>
-                        </v-flex>
-                        <v-flex xs9>
-                          <v-list-item-content>
-                            <v-list-item-title style="white-space: normal" v-html="item.title"/>
-                            <v-list-item-subtitle v-html="item.subtitle"/>
-                          </v-list-item-content>
-                        </v-flex>
-                        <v-flex xs1>
-                          <v-btn icon><v-icon small class="primary-color--text">mdi-close</v-icon></v-btn>
-                        </v-flex>
-                      </v-layout>
-                    </v-list-item>
-                  </v-list>
-                </v-card>
-              </v-flex>
-            </v-layout>
-          </v-container>
+          <v-divider
+            :inset="inset" 
+            class="my-0"/>
+          <div class="notifDrawerItems">
+            <div
+              v-for="(item, i) in items"
+              :key="i"
+              :id="'notifItem-'+i"
+              class="notifDrawerItem">
+              <div
+                class="notifContent"
+                @click="getActivityId('notifItem-'+i)"
+                v-html="item.content_notif"></div>
+                <span
+                :id="'notifClose-'+i" 
+                class="notifClose" 
+                @click="hideNotification('notifItem-'+i)"><i class="uiIconClose uiIconLightGray"></i></span>
+            </div>
+          </div>
+          <v-row class="notifFooterActions mx-0">
+            <v-card 
+              flat
+              tile 
+              class="d-flex flex justify-end mx-2">
+              <v-btn 
+                text
+                small
+                class="text-uppercase caption "
+                color="primary"
+                @click="notificationOperation(null, 'markAllAsRead')">{{ this.$t('homepage.topBar.notification.MarkAllAsRead') }}</v-btn>
```

Comment:
```suggestion
                @click="markAllAsRead()">{{ this.$t('homepage.topBar.notification.MarkAllAsRead') }}</v-btn>
```
- markAllAsRead method to be implemented

Reply:
Done!